components/homme/src/share/cxx/ExecSpaceDefs.cpp

```diff
@@ -136,21 +157,33 @@ team_num_threads_vectors_for_gpu (
     const int num_threads = ( (tp.max_threads_usable > num_device_threads) ?
                               num_device_threads :
                               tp.max_threads_usable );
+
+//printf("tp.prefer_threads: %4d %4d \n",num_threads,  prevpow2(num_device_threads / num_threads));
+
     return std::make_pair( num_threads,
                            prevpow2(num_device_threads / num_threads) );
   } else {
     const int num_vectors = prevpow2( (tp.max_vectors_usable > num_device_threads) ?
                                       num_device_threads :
                                       tp.max_vectors_usable );
+
+//printf("NOT tp. prefer_threads: %4d %4d \n",num_device_threads / num_vectors,
+//                           num_vectors);
+
     return std::make_pair( num_device_threads / num_vectors,
                            num_vectors );
   }
+#else  
+//manual override for HIP  
+return std::make_pair( 16,4 );
```

Comment:
Is this line meant to go in?

Reply:
yes. last time i looked i could not figure out what warps, etc. are for miXXX, so, it is not clear what the logic above would produce for the pair. this pair was defined from performance runs.